assets/js/modules/analytics/datastore/properties.js

```diff
@@ -138,49 +138,51 @@ const baseActions = {
 	 *
 	 * @param {string} propertyID Property ID to select.
 	 * @param {string} [internalPropertyID] Internal property ID (if available).
+	 * @return {void}
```

Comment:
This is actually a [`Generator`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Generator) instance now, so we can use this as the type.
```suggestion
	 * @return {Generator}
```